Formula/p/postgresql@15.rb

```diff
@@ -119,6 +123,29 @@ def post_install
     (var/"log").mkpath
     postgresql_datadir.mkpath
 
+    # Manually link files from keg to non-conflicting versioned directories in HOMEBREW_PREFIX.
+    %w[include lib share].each do |dir|
+      dst_dir = HOMEBREW_PREFIX/dir/name
+      src_dir = prefix/dir/"postgresql"
```

Comment:
With some src/dst examples: we should be able to handle this in Homebrew/brew for all these formulae with something like or by modifying https://github.com/Homebrew/brew/blob/cdf3964753b5f1132252742712c908e657ecc7a2/Library/Homebrew/keg.rb#L441-L442

Let me know if you need any additional help/pointers on that. Would really like to try to have the linking happen there instead.

Reply:
see other comment, this is basically copied from postgresql@17 formulae.